sandbox/sweep-reads.py

```diff
@@ -52,10 +52,10 @@ def fmt_fasta(name, seq, labels=[]):
         name=name, labels='\t'.join([str(l) for l in labels]), seq=seq)
 
 
-def fmt_fastq(name, seq, accuracy, labels=[]):
+def fmt_fastq(name, seq, quality, labels=[]):
```

Comment:
@ctb this script uses a ReadBufferManager which is incompatible with write_record.